src/app/containers/Brand/index.test.jsx

```diff
@@ -17,11 +20,38 @@ const newsServiceContextStub = {
     brandBackgroundColour: `${C_POSTBOX}`,
     brandLogoColour: `${C_WHITE}`,
   },
+  translations: {
+    skipLinkText: 'Skip to content',
+  },
+};
+
+const variantServiceContextStub = {
+  ...newsServiceContextStub,
+  script: latin,
+  variant: 'test',
+  scriptLinkVariants: {
+    test: {
+      scriptLinkText: 'Test',
+      scriptLinkOffscreenText: 'Test-variant',
+    },
+  },
+};
+
+jest.mock('#lib/utilities/variantHandler');
+const variantHandlers = require('#lib/utilities/variantHandler');
+
+variantHandlers.getOtherVariant.mockImplementation(() => 'test');
+
+const spy = jest.spyOn(cookies, 'setPreferredVariantCookie');
```

Comment:
While naming this `spy` would be appropriate if it was in close proximity to where it was being used. Scrolling down to where `spy` is used requires one to keep the information that it points to `setPreferredVariantCookie` in his/her head. Could we rename this to `setPreferredVariantCookieSpy`.

Reply:
Yes. Renamed this to `setPreferredVariantCookieSpy` as suggested.